src/ReactiveUI.Validation/ReactiveUI.Validation.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>MonoAndroid90;Xamarin.iOS10;Xamarin.Mac20;Xamarin.TVOS10;tizen40;netstandard2.0</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);net461;uap10.0.17763</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);net461</TargetFrameworks>
```

Comment:
Worth reverting this change once you're done as we still want the CI/CD to generate NuGet packages for `uap10.0.17763`

Reply:
Damn it! Thought I'd reverted that, sorry, will do now.